index.html

```diff
@@ -132,7 +135,7 @@ <h2>Sponsors</h2>
       </section>
 
 
-      <section>
+      <section id="footer">
         <div>
           <h3>Support</h3>
           <ul>
```

Comment:
The HTML is changed only to add ids. Nicely done! ID rather than class is a choice toward specificity but doesn't allow reusing styling in a flexible way. Are there areas where you wrote the same CSS for two different ids?